app/views/users/change_username.html.erb

```diff
@@ -1,50 +1,53 @@
+      <% end %>
+    </p>
+    <p>
+      <%= t(".more_info_html",
+            account_faq_link: link_to(t(".account_faq"), archive_faq_path("your-account", anchor: "namechange")),
+            contact_support_link: link_to(t(".contact_support"), new_feedback_report_path)) %>
+    </p>
+  </div>
+  <p class="note">
+    <%= t(".new_pseud_instead_html", create_a_new_pseud_link: link_to(t(".create_a_new_pseud"), new_user_pseud_path(@user))) %>
   </p>
-  <p>
-    <%= t(".more_info",
-          account_faq_link: (link_to t(".account_faq"), archive_faq_path("your-account", anchor: "namechange")),
-          contact_support_link: (link_to t(".contact_support"), new_feedback_report_path)
-        ).html_safe %>
-  </p>
-</div>
-<p class="note">
-  <%= ts("If that is not what you want, you can <a href=\"#{new_user_pseud_path(@user)}\">create a new Pseud</a> instead.").html_safe %>
-</p>
-<!--/descriptions-->
+  <!--/descriptions-->
 
-<!--subnav-->
-<%= render "edit_header_navigation" %>
-<!--/subnav-->
+  <!--subnav-->
+  <%= render "edit_header_navigation" %>
+  <!--/subnav-->
 
-<!--main content-->
+  <!--main content-->
 
-<%= form_tag changed_username_user_path(@user) do %>
-  <dl>
-    <dt><%= ts("Current user name") %></dt>
-    <dd><p class="important informational"><%= @user.login %></p></dd>
-    <dt><%= label_tag :new_login, ts("New user name") %></dt>
-    <dd>
-      <%= text_field_tag :new_login, @new_login, autocomplete: "off", "aria-describedby" => "new-login-field-description" %>
-      <p class="footnote" id="new-login-field-description">
-        <%= ts("%{minimum} to %{maximum} characters (A-Z, a-z, _, 0-9 only), no spaces, cannot begin or end with underscore (_)",
-               minimum: ArchiveConfig.LOGIN_LENGTH_MIN,
-               maximum: ArchiveConfig.LOGIN_LENGTH_MAX) %>
-      </p>
-    </dd>
-    <dt><%= label_tag :password, ts("Password") %></dt>
-    <dd><%= password_field_tag :password %></dd>
-    <dt class="landmark"><%= ts("Submit") %></dt>
-    <dd class="submit actions">
-      <%= submit_tag ts("Change User Name"), data: { confirm: ts("Are you sure you want to change your user name?") } %>
-    </dd>
-  </dl>
+  <%= form_tag changed_username_user_path(@user), autocomplete: "off" do %>
+    <dl>
+      <dt><%= t(".current_username") %></dt>
+      <dd><p class="important informational"><%= @user.login %></p></dd>
+      <dt><%= label_tag :new_login, t(".new_username") %></dt>
+      <dd>
+        <%= text_field_tag :new_login, @new_login, "aria-describedby": "new-login-field-description" %>
+        <p class="footnote" id="new-login-field-description">
+          <%= t(".username_requirements",
+                minimum: ArchiveConfig.LOGIN_LENGTH_MIN,
+                maximum: ArchiveConfig.LOGIN_LENGTH_MAX) %>
+        </p>
+      </dd>
+      <dt><%= label_tag :password, t(".password") %></dt>
+      <dd><%= password_field_tag :password, nil, autocomplete: "disabled" %></dd>
```

Comment:
Does this need `autocomplete: "disabled"` when the form has `autocomplete: "off"` because it's a password field?

Reply:
I think I'm confused. Why are we adding it if it's not a attribute that's available on the field?

Reply:
My bad, I got my wires crossed and forgot to remove this one